packages/cardano-services/src/Metadata/HttpMetadataService.ts

```diff
@@ -0,0 +1,52 @@
+import { Cardano, ProviderError, ProviderFailure } from '@cardano-sdk/core';
+import { ExtMetadataResponse, StakePoolExtMetadataService } from '../StakePool/types';
+import { Logger } from 'ts-log';
+import { ValidationError, validate } from 'jsonschema';
+import { getExtMetadataUrl, getSchemaFormat, loadJsonSchema } from './util';
+import axios, { AxiosInstance } from 'axios';
+
+const HTTP_CLIENT_TIMEOUT = 1 * 1000;
+const HTTP_CLIENT_MAX_CONTENT_LENGTH = 5000;
+
+export const createHttpMetadataService = (
```

Comment:
```suggestion
export const createHttpStakePoolExtMetadataService = (
```
And maybe let's move this to src/StakePool?

Reply:
Yeah, since I moved the `types.ts`, I was thinking to move the HttpMetadataService.ts + schemas folder as well and leaving in `/Metadata` only Txs-related ones ? @mkazlauskas

Reply:
Yep, sounds good

Reply:
Addressed in https://github.com/input-output-hk/cardano-js-sdk/pull/490/commits/edd4511f91ec1c009fb3bc17466618f44212aff8 @mkazlauskas